package/network/services/dnsmasq/files/dnsmasqsec.hotplug

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+TIMEVALIDFILE=/tmp/ntp.d/dnsmasqsec
+
+[ "$ACTION" = stratum ] || exit 0
```

Comment:
> In that case maybe only support step

Checking only for step does not sound right, as based on the process description there might not be a step event if the time is right enough at ntpd startup (e.g. due to a real-time clock in the device). At least that is how I read the algorithm description: https://git.busybox.net/busybox/tree/networking/ntpd.c#n85

And real-life behaviour matches that. Restarting sysntpd creates only a stratum event:
```
Thu Jun 16 17:46:11 2016 user.notice root: RESTART NTPD
Thu Jun 16 17:46:15 2016 user.notice ntpd: Stratum change, stratum=2 interval=1 offset=0.004251
```